device/dhcp/clients/v6/dhcpv6client.yaml

```diff
@@ -0,0 +1,195 @@
+components:
+  schemas:
+    Device.Dhcpv6client:
+      description: >-
+        Configuration for emulated DHCPv6 Client on a single Interface. If the DHCPv6 Client receives one or more 
+        DHCPv6 ADVERTISE messages from one or more servers then the client chooses one server from which to request 
+        configuration parameters, based on the configuration parameters offered by the server in the DHCPv6 ADVERTISE 
+        messages. If all configuration parameters match then the first server will be chosen.
+        https://www.rfc-editor.org/rfc/rfc8415.html
+      type: object
+      required: [name,client_identifier]
+      properties:
+        name:
+          x-include: ../../../../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        rapid_commit:
+          description: >-
+            If Rapid Commit is set, client initiates Rapid Commit two-message exchange by including Rapid Commit option 
+            in Solicit message.
+          type: boolean
+          default: false
+          x-field-uid: 2
+        client_identifier:
+          description: |-
+            The client identifier option is used to carry a DUID identifying a client between a client and a server.            
+          $ref: '../../common/dhcpv6options.yaml#/components/schemas/Dhcpv6Options.ClientIdentifier'
+          x-field-uid: 3
+        server_identifier:
```

Comment:
yes client if it knows the server id, will put that id so that it received responses from a particular server not all as the initial message is multicast